content/concepts/protocols.md

```diff
@@ -105,16 +105,8 @@ all of the registered match functions. If any returns `true`, the associated han
 This gives you a lot of flexibility to do your own "fuzzy matching" and define whatever rules for protocol matching
 make sense for your application.
 
-#### Match using semver
-
 If you'd like to concurrently support a range of numbered versions, you may want to use semantic versioning (aka [semver](https://semver.org)).
```

Comment:
Can we remove this sentence as well? We've moved away from sem-versioning protocols, since in practice it hasn't proved feasible to distinguish backwards-compatible and backwards-incompatible protocol changes.

Reply:
Ah I see, updated.